Source/Private/_ASPendingState.mm

```diff
@@ -100,7 +100,7 @@ @implementation _ASPendingState
   @package //Expose all ivars for ASDisplayNode to bypass getters for efficiency
 
   UIViewAutoresizing autoresizingMask;
-  unsigned int edgeAntialiasingMask;
+  CAEdgeAntialiasingMask edgeAntialiasingMask;
```

Comment:
@hanton not a blocking comment, but I think there are other places in which we also use `unsigned int` instead of `CAEdgeAntialiasingMask`? I'm totally down to change them too :)

Reply:
@nguyenhuy sorry for missing this comment :p

The PR for fixing this issue is ready at #1667